package.json

```diff
@@ -9,7 +9,7 @@
   ],
   "resolutions": {
     "webpack": "4.30.0",
-    "grpc": "1.20.2",
+    "grpc": "1.20.3",
```

Comment:
minor change, this works locally, and only is used locally (in dev)
the rest are submodule & yarnage from those submodules, they are merged so this is good to go